src/core_plugins/tile_map/public/editors/wms_options.js

```diff
@@ -0,0 +1,11 @@
+import { uiModules } from 'ui/modules';
+import wmsOptionsTemplate from './wms_options.html';
+const module = uiModules.get('kibana');
+
+module.directive('wmsOptions', function () {
```

Comment:
i agree with Tim, we could isolate this scope to make it easier to see whats going on

Reply:
+1